front/app/api/projects/types.ts

```diff
@@ -187,6 +187,7 @@ export interface IProjectAttributes {
     taking_poll: ActionDescriptor<PollDisabledReason>;
     annotating_document: ActionDescriptor<DocumentAnnotationDisabledReason>;
   };
+  uses_content_builder: boolean;
 }
```

Comment:
@luucvanderzee Is this ever set by the FE? Or is just determined/returned by the BE?

Reply:
@amanda-anderson completely determined by the BE